ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -657,6 +659,12 @@ object Config {
             s"Use the experimental High Availability feature with the indexer. Should not be used in production."
           )
           .action((_, config) => config.copy(enableHa = true))
+
+        opt[Unit]("use-error-codes-v2")
```

Comment:
Can we have a meaningful name for this command-line parameter, please? E.g., `use-self-service-error-codes` or `use-standard-error-codes` or similar.